server/entity/Media.ts

```diff
@@ -104,6 +108,18 @@ class Media {
   @Column({ type: 'datetime', nullable: true })
   public mediaAddedAt: Date;
 
+  @Column({ nullable: true })
+  public ratingKey?: string;
```

Comment:
We need a rating key to track the 4k content as well.

Reply:
Alright, I thought we didn't need this distinction, I'll test it out.

Reply:
I tried this with a 4k item and it seems to work. It isn't linking to the actual item video, just the initial media page. Can you explain when we would need it if I'm missing something.

Reply:
For people (like me) who have 4K movies in separate libraries. The rating key is different.

Reply:
Alright, I see.

Reply:
I was about to dev this feature, so ty @ankarhem  for your work 👍 
It seems to be already done, but +1 me if you need any help

Reply:
Yeah, need this extra key here for 4K support but after that it's done. I'm waiting for my other pr to be merged since that will require me to do some work here anyway to fix the migrations.